lib/core-server/src/__snapshots__/cra-ts-essentials_manager-dev

```diff
@@ -216,220 +216,3 @@ Object {
   ],
```

Comment:
This can be merged like this but I think it would be better to remove this file from the PR.